locales/en/messages.json

```
@@ -3930,16 +3930,16 @@
         "message": "Dynamic Notch Width Percent"
     },
     "pidTuningDynamicNotchQ": {
-        "message": "Dynamic Notch Q"
+        "message": "Q factor"
     },
     "pidTuningDynamicNotchMinHz": {
-        "message": "Dynamic Notch Min Frequency [Hz]"
+        "message": "Min Frequency [Hz]"
     },
     "pidTuningDynamicNotchMaxHz": {
-        "message": "Dynamic Notch Max Frequency [Hz]"
+        "message": "Max Frequency [Hz]"
     },
     "pidTuningDynamicNotchCount": {
-        "message": "Dynamic Notch Count"
+        "message": "Count"
```

```suggestion
        "message": "Notch Count"
```
To make it more clear what this "count" is about. But I'm fine with only "Count" if you think it's suited better.

---

OK will do